src/main/java/BackstagePassesUpdater.java

```diff
@@ -1,3 +1,5 @@
+import java.util.ArrayList;
```

Comment:
**question:** I am _extremely_ rusty on my Java so please do forgive me if I mention this in error, but does this import (and in a few other files) need to be here?

Reply:
It doesn't.  Will remove.